openstack_controller/datadog_checks/openstack_controller/api/api_rest.py

```diff
@@ -30,15 +30,20 @@ def has_admin_role(self):
     def component_in_catalog(self, component_types):
         return self._catalog.has_component(component_types)
 
-    def get_response_time(self, endpoint_types, remove_project_id=True):
+    def get_response_time(self, endpoint_types, remove_project_id=True, is_heat=False):
         endpoint = (
-            self._catalog.get_endpoint_by_type(endpoint_types).replace(self._current_project_id, "")
-            if self._current_project_id and remove_project_id
-            else self._catalog.get_endpoint_by_type(endpoint_types)
+            self._catalog.get_endpoint_by_type(endpoint_types).replace("/v1/" + self._current_project_id, "")
```

Comment:
```suggestion
            self._catalog.get_endpoint_by_type(endpoint_types).replace(f"/v1/{self._current_project_id}", "")
```
sorry forgot to comment here as well!

Reply:
fixed.